ToDo.js

```diff
@@ -0,0 +1,376 @@
+class ServerRequests {
+    columnsPath;
+    cardsPath;
+
+    constructor() {
+        this.columnsPath = `https://todosy.iwanicki.wtf/api/v1/todo-columns/`
+        this.cardsPath = `https://todosy.iwanicki.wtf/api/v1/todo-item/`
+    }
+
+    async getColumnsObj() {
+        const response = await fetch(this.columnsPath, {
+            method: "GET"
+        });
+        const columnsAll = await response.json()
+        return columnsAll
+    }
+
+    async getCardsObj(colId) {
+        const response = await fetch(this.cardsPath + colId, {
+            method: "GET"
+        });
+        const cardsAll = await response.json()
+        return cardsAll
+    }
+
+    async sendColumnName(name, uuid) {
+        await fetch(this.columnsPath + uuid, {
+            method: "PUT",
+            body: name
+        })
+    }
+
+    async deleteColumn(columnId) {
+        await fetch(this.columnsPath + columnId, {
+            method: "DELETE",
+        })
+    }
+
+    async postNewColumn() {
+        const columnData = new FormData()
+        columnData.append("name", "Column")
+        await fetch(this.columnsPath, {
+            method: "POST",
+            body: columnData
+        })
+    }
+
+    async postCard(parentId, cardData) {
+        await fetch(this.cardsPath + parentId, {
+            method: "POST",
+            body: cardData
+        })
+    }
+
+    async putCard(modifiedCardData, cardId) {
+        await fetch(this.cardsPath + cardId, {
+            method: "PUT",
+            body: modifiedCardData
+        })
+    }
+
+    async deleteCard(cardId) {
+        await fetch(this.cardsPath + cardId, {
+            method: "DELETE",
+        })
+    }
+
+}
```

Comment:
This is good approach, you extracted data logic into separate class that do only  API communications. 

Improvements:
- Split it into to services, todo-item, todo-column, you want to have one class responsible to handle one category, a rule of thumb is to follow the endpoint naming convention. If you are given endpoint like `/api/v1/auth`, `/api/v1/users` you will probably create 2 classes handling these endpoints: AuthHttpService, UsersHttpService (https://java-design-patterns.com/patterns/service-layer/#explanation)